Games/types/Mafia/roles/cards/CommitSeppuku.js

```diff
@@ -0,0 +1,28 @@
+const Card = require("../../Card");
+const { PRIORITY_KILL_DEFAULT } = require("../../const/Priority");
+
+module.exports = class CommitSeppuku extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Convert Player": {
+                states: ["Night"],
+                flags: ["voting"],
+                targets: { include: ["alive"], exclude: ["Mafia"] },
+                action: {
+                    labels: ["kill", "seppuku"],
+                    priority: PRIORITY_KILL_DEFAULT,
```

Comment:
this should be a conversion priority not a kill priority imo

what is the intended mechanic if someone is masoned, culted, yakked the same night?

Reply:
No specific intended order. I'm not sure what happened in em or what we want to happen.